.circleci/config.yml

```diff
@@ -504,27 +600,22 @@ workflows:
       - cpu_tests_py38
       - cpu_tests_py39
       - gpu_tests_151:
-          test_dir: tests/experimental
-      - gpu_tests_160:
-          test_dir: tests/experimental
-      - gpu_tests_171:
-          test_dir: tests/experimental
-      - gpu_tests_151:
-          test_dir: tests/nn
+          test_list_file: tests/ci_test_list_1.txt
       - gpu_tests_160:
-          test_dir: tests/nn
+          test_list_file: tests/ci_test_list_1.txt
```

Comment:
hmm, I'm a bit uneasy with a static list of tests, it's one more thing to remember and it does not feel super scalable over time ? would pytest flags help here, so that we have a small list of tests to maintain ? (-k fsdp, ..)

Reply:
I am certainly open to suggestions. -k doesn't seem to be able to give us an easy way to maintain different subsets. The list is not hard to maintain. It is just a file name pointer and there is check to ensure we don't miss any tests in the lists. Definitely let me know if there are better ways.

Reply:
-k would be kind of the same, but with key topics instead of the list of subtests, that's at least smaller to maintain ?
ie: `fsdp`, `adascale`, `sharded_ddp` and any test under any of these "topics" would always be up to date ?